simcon_project/conversation_templates/models/template_node_response.py

```diff
@@ -1,8 +1,9 @@
 from django.db import models
+import uuid
 
 
 class TemplateNodeResponse (models.Model):
```

Comment:
This should have a foreign key to TemplateResponse (called 'parent_template_response' with related name 'node_responses' or something like that). It should cascade on delete (If TemplateResponse goes so should the node responses under that Template response)